dockerfiles/che/Dockerfile

```diff
@@ -1,18 +1,37 @@
-# Copyright (c) 2012-2019 Red Hat, Inc.
+# Copyright (c) 2018-2022 Red Hat, Inc.
 # This program and the accompanying materials are made
 # available under the terms of the Eclipse Public License 2.0
 # which is available at https://www.eclipse.org/legal/epl-2.0/
 #
 # SPDX-License-Identifier: EPL-2.0
 #
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
 
-FROM docker.io/adoptopenjdk/openjdk11:jre-11.0.11_9
-ENV LANG=C.UTF-8
-RUN echo "%root ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers && rm -rf /tmp/*
-EXPOSE 8000 8080
+# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8-minimal
+FROM registry.access.redhat.com/ubi8-minimal:8.5-230
+USER root
+ENV CHE_HOME=/home/user/eclipse-che
+ENV JAVA_HOME=/usr/lib/jvm/jre
+RUN microdnf install java-11-openjdk-headless tar gzip shadow-utils findutils && \
```

Comment:
Could pin to a specific runtime rpm with `microdnf install java-11-openjdk-headless-11.0.14.0.9-2.el8_5`